src/configuration.ts

```diff
@@ -0,0 +1,76 @@
+import fs from 'fs';
+import path from 'path';
+import defaults from 'defaults-deep';
```

Comment:
should we use lodash with lodash.merge instead?

Reply:
we could use lodash defaultsDeep if we don't want to merge arrays. currently the only place we have arrays in the config is the mail options. but it would be ok IMHO if the platform config would override the array completely.
BTW 'defaults-deep' was used in the original code. I think it would be good to replace it anyway since it has vulnerabilities.

Reply:
Then lets switch over to lodash.

Reply:
Ok. Moved to lodash.deepMerge().